src/lib/navigation.ts

```diff
@@ -271,6 +271,15 @@ export const navigation: Navigation[] = [
           }
         ]
       },
+      {
+        title: 'Databaser',
+        children: [
+          {
+            title: 'Å lage en PostgreSQL database for et prosjekt',
```

Comment:
Gjerne bruk samme tittel her som på selve siden, altså:
"Hvordan lage en PostgreSQL database for et prosjekt"

Reply:
fixed